engine/src/test/java/io/graphqlcrud/SQLDataFetcherTest.java

```diff
@@ -56,24 +56,33 @@ public void testDataFetcher() throws Exception {
                     "  }\n" +
                     "}";
             String result1 = executeSQL(query1,graphQLSchema);
-            Assertions.assertEquals(result1,"SELECT SSN FROM PUBLIC.CUSTOMER");
+            Assertions.assertEquals(result1,"SELECT PUBLIC.CUSTOMER.SSN FROM PUBLIC.CUSTOMER");
 
             String query2 = "{\n" +
                     "  accounts {\n" +
                     "    ACCOUNT_ID\n" +
                     "  }\n" +
                     "}";
             String result2 = executeSQL(query2,graphQLSchema);
-            Assertions.assertEquals(result2,"SELECT ACCOUNT_ID FROM PUBLIC.ACCOUNT");
+            Assertions.assertEquals(result2,"SELECT PUBLIC.ACCOUNT.ACCOUNT_ID FROM PUBLIC.ACCOUNT");
 
             String query3 = "{\n" +
                     "  customer(SSN : \"CST01002\") {\n" +
                     "    SSN\n" +
                     "  }\n" +
                     "}";
             String result3 = executeSQL(query3,graphQLSchema);
-            Assertions.assertEquals(result3,"SELECT SSN FROM PUBLIC.CUSTOMER WHERE SSN = 'CST01002'");
+            Assertions.assertEquals(result3,"SELECT PUBLIC.CUSTOMER.SSN FROM PUBLIC.CUSTOMER WHERE SSN = 'CST01002'");
 
+            String query4 = "{\n" +
+                    "  customers {\n" +
+                    "    accounts {\n" +
+                    "      SSN\n" +
+                    "    }\n" +
```

Comment:
Here SSN is field in customer ( iknow it is also in accounts but just relation key), and accounts has their own field names for ex:
```
customers {
      SSN        
      accounts {
         TYPE,
          ACCOUNT_ID
       }
 }
```